res/settings.ui

```diff
@@ -40,19 +40,6 @@
   <property name="modal">
    <bool>false</bool>
   </property>
-  <widget class="QDialogButtonBox" name="save_cancel_dialog_button_box">
```

Comment:
You mentioned you may prefer having a Save/Cancel button. I personally think having the settings instantly be reflected is preferable. Try it out, and if not I'll add them back with save/cancel logic.